packages/core/src/addresses/redux/actionTypes.js

```diff
@@ -137,5 +137,37 @@ export const GET_DEFAULT_CONTACT_ADDRESS_REQUEST =
 export const GET_DEFAULT_CONTACT_ADDRESS_SUCCESS =
   '@farfetch/blackout-core/GET_DEFAULT_CONTACT_ADDRESS_SUCCESS';
 
+/**
+ * Action type dispatched when delete default billing address request fails.
+ */
+export const DELETE_DEFAULT_BILLING_ADDRESS_FAILURE =
+  '@farfetch/blackout-redux/DELETE_DEFAULT_BILLING_ADDRESS_FAILURE';
```

Comment:
Check all action types values to make sure they reference blackout-core instead of blackout-redux, please

```suggestion
  '@farfetch/blackout-core/DELETE_DEFAULT_BILLING_ADDRESS_FAILURE';
```

Reply:
Done 👍🏻